.yarn/patches/jest-environment-jsdom-npm-29.5.0-fc600add1e.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/build/index.js b/build/index.js
+index ec8a133f3fbea56f4b395c96ca2b27bd4903c62e..d9050e1c504c2a487ab1debff44c3737071edbc5 100644
+--- a/build/index.js
++++ b/build/index.js
+@@ -92,6 +92,8 @@ class JSDOMEnvironment {
+ 
+     // TODO: remove this ASAP, but it currently causes tests to run really slow
+     global.Buffer = Buffer;
++    global.TextEncoder = TextEncoder;
++    global.TextEncoder = TextEncoder;
```

Comment:
```suggestion
```
why assign twice?

Reply:
good catch
```suggestion
+    global.TextDecoder = TextDecoder;
```

Reply:
what's the purpose of this patch btw? why do we need to override TextDecoder, if it works without it?